src/transformers/integrations.py

```diff
@@ -1299,6 +1305,115 @@ def on_train_end(self, args, state, control, **kwargs):
             self.tracker.stop()
 
 
+class ClearMLCallback(TrainerCallback):
+    """
+    A [`TrainerCallback`] that sends the logs to [ClearML](https://clear.ml/).
+
+    Environment:
+            CLEARML_PROJECT (`str`, *optional*, defaults to "HuggingFace Transformers"):
+                ClearML project name. If not set deafault project name will be assigned.
+            CLEARML_TASK (`str`, *optional* defaults to `"Trainer"`):
+                ClearML task name. If not set default task name will be assigned.
+    """
+
+    def __init__(self):
+        has_clearml = is_clearml_available()
+        if not has_clearml:
+            raise RuntimeError("ClearMLCallback requires 'clearml' to be installed. Run `pip install clearml`.")
+        if has_clearml:
+            import clearml
+
+            self._clearml = clearml
+        else:
```

Comment:
This would be impossible to reach: you check first `if not has_clearml` then followed by `if has_clearml` which means you already cover both cases. The else will never be triggered.

Please reformat this to be a single if else statement on the value of `has_clearml`

Reply:
Thank you so much for noticing this. I missed this fact completely. I have updated it to a single if-else statement,